cmsplugin_filer_image/templates/cmsplugin_filer_image/image.html

```diff
@@ -1,46 +1,50 @@
-{% load thumbnail filer_tags filer_image_tags sekizai_tags %}{% spaceless %}
-{% addtoblock "css" %}<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}cmsplugin_filer_image/css/filer_image.css" media="screen, projection" />{% endaddtoblock "css" %}
+{% load thumbnail filer_tags filer_image_tags sekizai_tags %}
+{% spaceless %}
+{% addtoblock "css" %}
+<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}cmsplugin_filer_image/css/filer_image.css" media="screen, projection"/>
+{% endaddtoblock %}
 
-{% with d1='0123456789'|make_list|random d2='0123456789'|make_list|random d3='0123456789'|make_list|random d4='0123456789'|make_list|random %}
-  {% with rnd='imagePlugin_'|add:d1|add:d2|add:d3|add:d4 %}
+{% addtoblock "js" %}
+{% if event_tracking %}
+<script type="text/javascript">
+ (function($) {
+   $(document).ready(function(){
+     var event = ['_trackEvent', '{{ event_tracking.category }}', '{{ event_tracking.action}}', '{{ event_tracking.label }}'];
+     $("#{{ rnd }} .image-link").off('click').click(function() {
+       _gaq.push(event);
```

Comment:
This relies on having _gaq set in the template but the user might not GA set up and this will throw an error. You should check:
````javascript
if ( _gaq && _gaq instanceof Array ){
//....
}
````